PPI-projektas/ClientApp/src/components/mainDiv/Notes/NoteDisplayElement.js

```diff
@@ -0,0 +1,20 @@
+﻿import React, {Component} from 'react'
+import './NoteDisplayElement.css'
+
+export class NoteDisplayElement extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    handleOpenNote = () => {
+        this.props.openNote(this.props.noteId);
+    }
```

Comment:
what is the use of this function? Why can't you call this.props.openNote directly in onClick?